### DISCUSSION TITLE
Make a build based on Blender LTS branch

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#2679] [General]
.

**Comment by ReinerBforartists** [comment]
This would be a second independant fork. I don't have the time for it, sorry  :)

**Comment by Draise14** [comment]
Yeah... It's not future proof and a lot of work to maintain. 

Once Blender official is released after Bcon3, the master is about as stable as the LTS. Python add-ons would then be up to date with an official release. This is the "LTS" version of Bforartists which usually have a shelf life of 6 to 12 month. User practice for LTS is stick to that version another year or two. 

It's branching away from the master by two steps.... Which is not easy and not worth it as it has a shelf life. Forking from master is only 1 step away and future proof.

**Comment by ghost** [comment]
ok then just an idea :)

**Comment by Draise14** [comment]
Yeah! I have had some users on discord ask about it. But I give that explanation and all good.
